### ISSUE TITLE
Pownce URL is no longer resolving

### ISSUE THREAD

**Issue by riddle** [iamcal/oembed#65]
http://oembed.com

```
<p>A <i>consumer</i> (e.g. <a href="http://pownce.com/">Pownce</a>) makes the following HTTP request:</p>
```

Obviously not a big deal, just thought I would mention it seeing as Pownce closed down almost 6 years ago. :)

**Comment by tschoffelen** [comment]
This should be closed, is fixed in PR #75.

**Comment by iamcal** [comment]
oops, thanks
